macros/WebRTCSidebar.ejs

```diff
@@ -79,6 +79,7 @@ var text = mdn.localStringMap({
         <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/adapter.js"><%=text['adapter_js']%></a></li>
         <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/Taking_still_photos"><%=text['still_photos']%></a></li>
         <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/Simple_RTCDataChannel_sample"><%=text['simple_sample']%></a></li>
+        <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/build_a_phone_with_peerjs"><%=text['build_a_phone_with_peer_js']%></a></li>
```

Comment:
We use sentence case for slugs and headings in documents, so in the first case `build_a_phone_with_peerjs` should be updated to `Build_a_phone_with_peerjs`.

Also for this to work you'll have to add an quivalent entry into the `'en-US': { ... }` object. A single line along the lines of `'build_a_phone_with_peer_js': 'Build a phone with peer js',` inside the `en-US` string list would work fine.